### DISCUSSION TITLE
Tabular Editor 3 blocking dlls even after the script has been run

### DISCUSSION THREAD

**Discussion by bernatagulloesbrina** [TabularEditor/TabularEditor3#1104] [Ideas]
Hi, 
I was trying again to play with custom dlls, and I see that Tabular Editor still blocks the file after it has executed the script, so you can't modify the dll unless you shut down tabular editor, which makes any development extremely difficult. I know it might not be on top of your priorities, but if it's like a like of code somewhere, it would be a nice improvement. 

Severity	Code	Description	Project	File	Line	Suppression State
Warning		Could not copy "obj\Debug\ExtendedTOMWrapper.dll" to "bin\Debug\ExtendedTOMWrapper.dll". Beginning retry 4 in 1000ms. The process cannot access the file 'bin\Debug\ExtendedTOMWrapper.dll' because it is being used by another process. The file is locked by: "TabularEditor3 (118560)"	ExtendedTOMWrapper

**Comment by otykier** [comment]
Thanks for reporting - we'll look into it
